packages/core/admin/admin/src/pages/SettingsPage/pages/Webhooks/EditView/components/WebhookForm/utils/getWebhookValidationSchema.js

```diff
@@ -0,0 +1,62 @@
+import * as yup from 'yup';
+import { translatedErrors } from '@strapi/helper-plugin';
+
+const NAME_REGEX = /(^$)|(^[A-Za-z][_0-9A-Za-z ]*$)/;
+const URL_REGEX = /(^$)|((https?:\/\/.*)(d*)\/?(.*))/;
+
+export const getWebhookValidationSchema = ({ formatMessage }) =>
```

Comment:
```suggestion
export const makeWebhookValidationSchema = ({ formatMessage }) =>
```
I think make is better than get since get normally infers data and you're basically doing a factory here?